site/formulas/decoding.md

```diff
@@ -1 +1,42 @@
 # Decoding to Extended Coordinates
+
+Given 32 octets of data, `s_bytes`, representing a compressed
+Ristretto point, the implementation must:
+
+1. Check that `s_bytes` is in fact 32 octets.  (In typed languages the
+   implementor should guarantee this through the type system, otherwise
+   there should be an explicit check.)
```

Comment:
I think step 1 is included in step 2... the canonical encoding of a field element for `ristretto255` is 32 bytes, so checking the length is included in checking that, and we can note that in the instructions on how to check whether the encoding is canonical.